packages/client/components/BottomControlBarTips.tsx

```diff
@@ -80,15 +80,25 @@ const EstimateHelpMenu = lazyPreload(
   async () => import(/* webpackChunkName: 'EstimateHelpMenu' */ './MeetingHelp/EstimateHelpMenu')
 )
 
-const demoHelps = {
+const demoHelps: Record<NewMeetingPhaseTypeEnum, LazyExoticPreload<any> | null> = {
   checkin: DemoReflectHelpMenu,
   reflect: DemoReflectHelpMenu,
   group: DemoGroupHelpMenu,
   vote: DemoVoteHelpMenu,
-  discuss: DemoDiscussHelpMenu
-} as Record<NewMeetingPhaseTypeEnum, LazyExoticPreload<any>>
+  discuss: DemoDiscussHelpMenu,
+  ESTIMATE: null,
```

Comment:
+1 You could type it as `Partial<Record<...>>` to avoid having to specify all the empty values.

Reply:
yeah, but his way it's will scream if a new phase is added and not handled here, which is good, imo

Reply:
@Dschoordsch after giving it a second thought, you're right. i thought it'd be a good reminder to add/not add a tooltip but this should be considered on the design phase, not enforced by TS. will change.